### PR TITLE
Validation: Fixed duplicate descriptions  

### PR DESCRIPTION
**Pull request by annmariehitchcock**
This PR addresses the following validation issues: 
duplicate descriptions

### CODE REVIEW

File: api-reference/beta/resources/ediscovery-datasource.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: "dataSource resource type"
-description: "The dataSource entity is an abstract base class used to identify sources of content for eDiscovery."
+title: The dataSource resource type
```
Comment (Lauragra) [inline api-reference/beta/resources/ediscovery-datasource.md:2]: Please remove the article "The" from the title attribute and the H1 heading.

File: api-reference/beta/resources/ediscovery-datasourcecontainer.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-title: "dataSourceContainer resource type"
-description: "Base class for Custodians and Non-Custodial data sources."
+title: The dataSourceContainer resource type
```
Comment (Lauragra) [inline api-reference/beta/resources/ediscovery-datasourcecontainer.md:2]: Global: Please remove the article The from all instances where it was added.

**Comment by annmariehitchcock** [comment]
**Global effort to fix build validation errors**
 @Lauragra 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Comment by annmariehitchcock** [comment]
Descriptions have been revised. Please review, thank you!

**Comment by annmariehitchcock** [comment]
#feedback-addressed

**Comment by annmariehitchcock** [comment]
Description, heading and title issues have been resolved. Please re-review. Thank you!

**Pull request closed without merging**